assets/scss/modules/_tables.scss

```diff
@@ -1 +1,117 @@
+////
+///	Tables
+/// @group Tables
+/// @author gbaamonde@makingsense.com
+////
+/// Table Component -
+/// This is a simple easy-to-use table component.
+/// @example markup
+///   <table class="dp-c-table">
+///     <caption>asdasdasd</caption>
+///     <thead>
+///       <tr>
+///         <th>
+///           Column 1
+///         </th>
+///         <th>
+///           Column 2
+///         </th>
+///       </tr>
+///     </thead>
+///     <tfoot>
+///       <tr>
+///         <td colspan="2">
+///           Single column for <tfoot>
+///         </td>
+///       </tr>
+///     </tfoot>
+///     <tbody>
+///       <tr>
+///         <td>
+///           Content 1
+///         </td>
+///         <td>
+///           Content 2
+///         </td>
+///       </tr>
+///     </tbody>
+///   </table>
 
+@mixin dp-c-table {
```

Comment:
Why use a mixin here instead of a simple class? I'm asking out of curiosity,

Reply:
@cbernat  as we chat, the mixins can receive parameters and arguments, allows us to dynamising our css, while the loose classes do not. This allows us to reuse our code.